app/application/interfaces/repositories.py

```
@@ -962,3 +986,50 @@ async def count(
         to_timestamp: datetime | None = None,
     ) -> int:
         """Return total count of audit log entries for tenant with optional filters."""
+
+
+# Webhook subscription repository interface
+class IWebhookSubscriptionRepository(Protocol):
+    """Protocol for webhook subscription repository (event push)."""
+
+    async def get_active_by_tenant(
+        self, tenant_id: str
+    ) -> list["WebhookSubscriptionForDispatch"]:
```

Ruff (0.15.5)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)